extensions/vscode/webviews/homeView/src/stores/home.ts

```diff
@@ -232,6 +232,111 @@ export const useHomeStore = defineStore("home", () => {
       .length;
   });
 
+  // Not considered an alert at this time.
+  const isNotPythonProject = computed(() => {
+    return !pythonProject.value;
+  });
+
+  const alertPythonEmptyRequirements = computed(() => {
```

Comment:
The `alert` prefix on a few of these I found a bit strange. I think it makes sense for the aggregate ones

Reply:
refactored with the re-organization with the store.